.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,172 @@
+name: C++ CI
```

Comment:
The `name` and file name (`c-cpp.yml`) are a bit strange, any better idea?

Reply:
Probably just call it `CI`?

Reply:
Those were left over from the Actions template that I used when creating the file. I agree though, they should be changed.